index.js

```diff
@@ -320,56 +328,82 @@ Color.prototype = {
 	},
 
 	clone: function () {
-		var col = new Color();
-		col.values = clone(col.values);
-		return col;
+		// NOTE(SB): using node-clone creates a dependency to Buffer when using browserify,
+		// making the final build way to big to embed in Chart.js. So let's do it manually,
```

Comment:
This change will not impact the Chart.js minified size because Chart.js uses an old version of *color* which [doesn't use node-clone](https://github.com/chartjs/color/blob/master/src/color.js#L311).